tasks/gen/templates/model_template.cr

```diff
@@ -8,7 +8,7 @@ class Lucky::ModelTemplate < Teeplate::FileTree
   directory "#{__DIR__}/model/"
 
   def initialize(@name : String)
-    @pluralized_name  = LuckyInflector::Inflector.pluralize(@name)
+    @pluralized_name = LuckyInflector::Inflector.pluralize(@name)
```

Comment:
I think we can do without this change.

Reply:
This was done automatically by `crystal tool format`